pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -32,7 +32,7 @@ import (
 	tracingapi "k8s.io/component-base/tracing/api/v1"
 	"k8s.io/kubernetes/pkg/features"
 	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
-	"k8s.io/kubernetes/pkg/kubelet/images"
+	imagepullmanager "k8s.io/kubernetes/pkg/kubelet/images/pullmanager"
```

Comment:
nit: Comment for this  whole commit: Can we move it where it is first introduced?

Reply:
I would like to keep this move as a separate commit if you don't mind. I feel like moving some of the bits that were previously available to all into a separate package might be worth a commit on its own and makes the flow somewhat easier to follow.